<ui version="4.0" >
 <class>proxy</class>
 <widget class="QWidget" name="proxy" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Proxy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QCheckBox" name="kcfg_isProxyEnabled" >
     <property name="text" >
      <string>Enable proxy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Proxy Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6" >
      <item row="1" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Type:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QComboBox" name="kcfg_proxyType" >
        <item>
         <property name="text" >
          <string>SOCKS 5</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>HTTP</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Host:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="KLineEdit" name="kcfg_proxyHostName" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Port:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="kcfg_proxyPort" >
        <property name="maximum" >
         <number>10000</number>
        </property>
        <property name="value" >
         <number>1080</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>293</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>Username:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2" >
       <widget class="KLineEdit" name="kcfg_proxyUserName" />
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_13" >
        <property name="text" >
         <string>Password:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2" >
       <widget class="KLineEdit" name="kcfg_proxyPassword" />
      </item>
      <item row="6" column="0" >
       <spacer name="verticalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
